README.md

```diff
@@ -20,6 +20,12 @@ docker run -it -p 127.0.0.1:8080:8080 -v "$PWD:/home/coder/project" codercom/cod
 
 ## Getting Started
 
+### Installation
+
+```bash
+curl -s https://raw.githubusercontent.com/cdr/code-server/issue-1396/install_helper.sh | bash -s
```

Comment:
Why the `-s` for `bash`?

Reply:
Specifies input should come from stdin-- it's possible it can be omitted

Reply:
Yea `-s` is only required if you want to pass in args.